### PR TITLE
fixed text measurement regression

### PR DESCRIPTION
**Pull request by kevinvangelder**
When we don't round up the text always gets clipped.

### CODE REVIEW

File: ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs [hunk 1]
```diff
@@ -213,8 +213,8 @@ private static YogaSize MeasureText(ReactTextShadowNode textNode, YogaNode node,
                 var normalizedHeight = YogaConstants.IsUndefined(height) ? double.PositiveInfinity : height;
                 textBlock.Measure(new Size(normalizedWidth, normalizedHeight));
                 return MeasureOutput.Make(
-                    (float)textBlock.DesiredSize.Width,
-                    (float)textBlock.DesiredSize.Height);
+                    (float)Math.Ceiling(textBlock.DesiredSize.Width),
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs]: >Ceiling [](start = 32, length = 7)

That's strange that cropping occurs on WPF and not on UWP.  We were going to use the same technique for UWP, but fixed the rounding issue in Yoga.  [https://github.com/facebook/yoga/pull/300](https://github.com/facebook/yoga/pull/300)

Comment (matthargett) [inline ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs]: does that mean we should publish a new package (same hash they published to npm/cocoapods), and then update our version reference to fix the issue?

Comment (rozele) [inline ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs]: I think this is also a problem on UWP still too.  Can you add the Math.Ceiling to UWP as well?

**Comment by rozele** [comment]
:shipit:

**Pull request merged**